documentation/guides/consumer/basic_setup.md

```diff
@@ -33,5 +35,5 @@ end
 - durable topic `example_exchange` exchange created or redeclared
 - durable `example_queue` queue created or redeclared and bound to `example_exchange` exchange
 - queue `example_queue` has a deadletter exchange set to `example_exchange.deadletter`
-- every `handle_message` callback will be executed in a separate process
+- every `handle_message` callback will be executed in a separate process (supervised task)
```

Comment:
```suggestion
- every `handle_message` callback will be executed in a separate process ([supervised task](https://hexdocs.pm/elixir/1.10.3/Task.html#module-supervised-tasks))
```

Reply:
Added a suggestion below with the link provided by Joel. 
This would have to be added to all the guide .md files though.

Reply:
Done for all guides